action.yml

```diff
@@ -45,32 +45,30 @@ runs:
         exec &> >(tee -a "analyze_logs.txt")
         fossa analyze
 
-      # We only want to run license compliance test if FOSSA scan succeeds. This is to unblock CI
+      # We only want to run license compliance test if `fossa test` succeeds. This is to unblock CI
       # on FOSSA outages.
     - if: steps.analyze.outcome == 'success'
-      name: 'Run FOSSA Test'
+      name: 'Run `fossa test`'
       id: test
       continue-on-error: true
       env:
         FOSSA_API_KEY: ${{ steps.set_key.outputs.key }}
       shell: bash
       run: |
         exec &> >(tee -a "test_logs.txt")
-        # Set timeout to 10 minutes
-        fossa test --timeout 600
+        # Set timeout to 5 minutes
+        fossa test --timeout 300
 
     - if: steps.analyze.outcome == 'failure' || steps.test.outcome == 'failure'
-      name: 'Send error to Sentry on FOSSA scan failure'
+      name: 'Send error to Sentry on `fossa-cli` errors'
       shell: bash
       env:
-        SENTRY_URL: https://self-hosted.getsentry.net/
-        SENTRY_ORG: self-hosted
-        SENTRY_DSN: https://0bc733fd07014f73a703d97ab5452ae2@self-hosted.getsentry.net/4
+        SENTRY_DSN: https://decbca863c554db095624ede8a83310c@o1.ingest.sentry.io/4505031352713216
       run: |
         if [[ ${{ steps.analyze.outcome }} == 'failure' ]]; then
           curl -sL https://sentry.io/get-cli/ | sh
           # Environment variables will automatically be sent, so we just want some minimal information
-          sentry-cli send-event -m "FOSSA scan failure" -t repo:$GITHUB_REPOSITORY -e url:$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID --logfile analyze_logs.txt
+          sentry-cli send-event -m "FOSSA analyze error" -t repo:$GITHUB_REPOSITORY -e url:$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID --logfile analyze_logs.txt
```

Comment:
Is there any simple parsing we can do on the logs.txt to get a more fine-grained error message than "FOSSA {analyze,test} error"? As currently implemented we're basically going to only have two issues in the Sentry project that accumulate all new errors forever (right?). I'd like to see us be able to distinguish better.

Reply:
Yep there is! Implemented this. Examples:
https://sentry.sentry.io/issues/4110296732/?project=4505031352713216&query=is%3Aunresolved&referrer=issue-stream&stream_index=0
https://sentry.sentry.io/issues/4110274496/?project=4505031352713216&query=is%3Aunresolved&referrer=issue-stream&stream_index=1

Reply:
and good point, it's good to have these errors separated out so it's easier to find out what type of errors are coming in